docs/connect.rst

```diff
@@ -0,0 +1,52 @@
+.. _schemaregistry_kafka_connect:
+
+Kafka Connect
+=============
+
+Kafka Connect and Schema Registry integrate to capture schema information from connectors. :ref:`Kafka Connect converters <connect_converters>`
+provide a mechanism for converting data from the internal data types use by Kafka Connect to data types represented as Avro.
+The AvroConverter will automatically register schemas generated by Source connectors. Sink Connectors will receive schema
+information in addition to the data for each message. This allows Sink connectors to know the structure of the data to provide
+additional capabilities like maintaining a database table structure or creating a search index. The AvroConverter will
+convert existing Avro data to the internal data types used by Kafka Connect.
+
+
+Example Configuration
+---------------------
+
+Configuring Kafka Connect to use the schema registry requires the user to change the ``key.converter`` or ``value.converter``
+properties in the :ref:`Connect worker configuration <connect_configuring_workers>`. The ``key.converter`` and ``value.converter``
+properties can be configured independently of each other.
+
+.. sourcecode:: properties
+
+    key.converter=io.confluent.connect.avro.AvroConverter
+    key.converter.schema.registry.url=http://schema-registry:8081
```

Comment:
Do we want to use in the example `localhost` instead, since it's going to be useful for quick copy pasting and matches what we have in `etc/schema-registry/connect-avro-distributed.properties`? (adding a note that one should substitute with the actual schema-registry hostname). The SR quickstart uses `localhost` anyways.

Or we could have `<schema-registry-hostname>` to show that this needs to be replaced in every case? 
Just a thought.